redisinsight/ui/src/pages/home/components/databases-list/DatabasesList.config.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+
+import { Instance } from 'uiSrc/slices/interfaces'
+import {
+  ColumnDef,
+  SortingState,
+  Table,
+} from 'uiSrc/components/base/layout/table'
+import {
+  BrowserStorageItem,
+  COLUMN_FIELD_NAME_MAP,
+  DatabaseListColumn,
+  DEFAULT_SORT,
+} from 'uiSrc/constants'
+import { localStorageService } from 'uiSrc/services'
+
+import DatabasesListCellName from './components/DatabasesListCellName/DatabasesListCellName'
+import DatabasesListCellHost from './components/DatabasesListCellHost/DatabasesListCellHost'
+import DatabasesListCellConnectionType from './components/DatabasesListCellConnectionType/DatabasesListCellConnectionType'
+import DatabasesListCellLastConnection from './components/DatabasesListCellLastConnection/DatabasesListCellLastConnection'
+import DatabasesListCellModules from './components/DatabasesListCellModules/DatabasesListCellModules'
+import DatabasesListCellControls from './components/DatabasesListCellControls/DatabasesListCellControls'
+import DatabasesListCellTags from './components/DatabasesListCellTags/DatabasesListCellTags'
+import { TagsCellHeader } from '../tags-cell/TagsCellHeader'
+
+export const SELECT_COL_ID = 'select-col-db'
+
+export const ENABLE_PAGINATION_COUNT = 15
```

Comment:
nit: It's not a boolean, so we don't enable anything. And we're not counting anything either.

```suggestion
export const PAGINATION_ITEMS = 15
export const PAGE_SIZE = 15
```